apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -103,10 +113,24 @@ schema() ->
                 ?IMPORTANCE_MEDIUM,
                 ?DESC(shared_subs),
                 #{}
+            )},
+        {mq_states,
+            db_schema(
+                [builtin_raft, builtin_local],
+                ?IMPORTANCE_MEDIUM,
+                ?DESC(mq_states),
+                #{}
+            )},
+        {mq_messages,
+            db_schema(
+                [builtin_raft, builtin_local],
+                ?IMPORTANCE_MEDIUM,
+                ?DESC(mq_messages),
+                #{}
             )}
-    ] ++ emqx_schema_hooks:list_injection_point('durable_storage', []).
```

Comment:
We cannot actually inject anything meaningful here, without private schema and config helpers.